### PR TITLE
Fix: Duplicated proxies

### PR DESCRIPTION
**Pull request by tuul-wq**
- Fix duplicated proxies in `ProxiesList`
- Added groups by chain to reduce work in UI

### CODE REVIEW

**Comment by tuul-wq** [comment]
Task linked: [CU-8693mw54k Wallet details. App is stuck when open details](https://app.clickup.com/t/8693mw54k)

**Review by egor0798 — APPROVED** [review]

File: src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts [hunk 1]
```diff
@@ -134,37 +129,74 @@ const $signatoryWallets = combine(
...
+      }
+
+      return acc;
+    }, []);
+
+    const chainsMap = Object.keys(chains).reduce<Record<ChainId, ProxyAccount[]>>((acc, chainId) => {
+      acc[chainId as ChainId] = [];
+
+      return acc;
+    }, {});
```
Comment (Asmadek) [inline src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts:151]: Maybe use `dictionary` here?

File: src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts [hunk 2]
```diff
@@ -134,37 +129,74 @@ const $signatoryWallets = combine(
...
+
+    return proxyUtils.sortAccountsByProxyType(proxiesForAccounts).reduce((acc, proxy) => {
+      acc[proxy.chainId].push(proxy);
+
+      return acc;
+    }, chainsMap);
```
Comment (Asmadek) [inline src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts:157]: `groupBy` maybe used here

**Review by Asmadek — APPROVED** [review]

File: src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts (hunk 2 above)
Comment (tuul-wq) [reply src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts:157]: I need to preserve original chains order

**Pull request merged**